auth_azure_ropc/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,5 @@
+This module add the possibility to login with Microsoft Azure ROPC flow
+
+https://learn.microsoft.com/en-us/azure/active-directory/develop/v2-oauth-ropc
+
+Microsoft recommends you do not use the ROPC flow. In most scenarios, more secure alternatives are available and recommended. This flow requires a very high degree of trust in the application, and carries risks that are not present in other flows. You should only use this flow when other more secure flows aren't viable.
```

Comment:
Actually, I think this is not specific to Azure. This looks like oauth2 Resource [Owner Password Credentials Grant](https://datatracker.ietf.org/doc/html/rfc6749#section-4.3) (aka Direct Access Grant in keycloak).

So you could perhaps rename it to `auth_oauth_ropc`, and give the Azure config as an example?

Also it might be worth mentioning why it is necessary in the README (i.e. the Odoo mobile app as you explained to me :)

Reply:
@sbidoul Done